### PR TITLE
added atoms for Button,Tabs,Toggle,Select and Input | added molecules for Tab Group , Privacy and Add Option

### PR DESCRIPTION
**Pull request by EmanFouad**
**Description:**

 **Added atoms styles for :**
  * Primary large ( default , hover , focus ) button
  * Active Tab
  * Inactive Tab
  * Toggles default ( ON , OFF )
  *  Text input no label ( default , hover , focus )
  
**Added molecules styles for :**
  * Tab Group
  * Privacy
  * Add Option

### CODE REVIEW

**Comment by moatazalaa14** [comment]
It's great work Eman 
Please consider:

- Group tab is not identical with design (font color , icon color and border for group are not identical)
- Adding focus state style for input 
- edit radio button in inactive tab group because it is not like design

**Comment by EmanFouad** [comment]
The required changes have been implemented

File: public/index.html [hunk 1]
```diff
@@ -2411,6 +2411,102 @@ <h3 class="text-lg font-bold text-white bg-dark text-center">AVATARS</h3>
...
+            Button Text
+        </button>
+        <button class="border-grey border px-4 py-2 ml-2 rounded-full flex justify-between items-center focus:outline-none">
+          <span class="p-2 rounded-xl bg-primary flex items-center">
+              <span class="p-1 rounded-lg  bg-white"></span>
+          </span>
+          <svg class="text-dark-grey w-4 ml-2" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 20 20" fill="currentColor">
+              <path fill-rule="evenodd" d="M4 3a2 2 0 00-2 2v10a2 2 0 002 2h12a2 2 0 002-2V5a2 2 0 00-2-2H4zm12 12H4l4-8 3 6 2-4 3 6z" clip-rule="evenodd" />
+          </svg>
+          <p class="text-dark-grey pl-2">Label</p>
```
Comment (moatazalaa14) [inline public/index.html]: still, group tab is not identical with design (Icon , font weight and spacing between them)

File: public/index.html [hunk 2]
```diff
@@ -2411,6 +2411,102 @@ <h3 class="text-lg font-bold text-white bg-dark text-center">AVATARS</h3>
...
+        </button>
+        <button class="bg-grey-bg px-4 py-2 ml-2 rounded-full flex justify-between items-center focus:outline-none">
+          <span class="p-2 rounded-xl bg-white border border-grey-shd4"></span>
+          <svg class="text-dark-grey w-4 ml-2" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 20 20" fill="currentColor">
+              <path fill-rule="evenodd" d="M4 3a2 2 0 00-2 2v10a2 2 0 002 2h12a2 2 0 002-2V5a2 2 0 00-2-2H4zm12 12H4l4-8 3 6 2-4 3 6z" clip-rule="evenodd" />
+          </svg>
+          <p class="text-dark-grey pl-2">Label</p>
+        </button>
+        <input class="border-grey-shd5 hover:border-grey-shd2 focus:outline-focus-border focus:outline-none focus:placeholder-dark focus:border-dark border rounded-md px-3 py-2 text-grey-sh01" type="text" placeholder="Enter Text" />
```
Comment (moatazalaa14) [inline public/index.html]: Edit focus state style for input

File: public/index.html [hunk 3]
```diff
@@ -2411,6 +2411,102 @@ <h3 class="text-lg font-bold text-white bg-dark text-center">AVATARS</h3>
...
+          <option value="public" selected>Public</option>
+          <option value="friends">Friends</option>
+          <option value="private">Private</option>
+        </select>
+      </div>
+    </section>
+    <section class="moleculse m-5">
+      <h2 class="text-3xl border-b-2 pb-3 mb-4">Moleculses</h2>
+      <div class="flex justify-between items-center">
+        <h3 class="w-2/3">Tap Group</h3>
```
Comment (moatazalaa14) [inline public/index.html]: Edit all group tabs like a design

**Review by moatazalaa14 — DISMISSED** [review]
Nice work Eman!

please consider: 

1. Edit notes in tab group and input
2. Make PR title more descriptive and put description for your work

File: public/index.html (hunk 1 above)
Comment (EmanFouad) [reply public/index.html]: Done

File: public/index.html (hunk 2 above)
Comment (EmanFouad) [reply public/index.html]: Done

File: package.json [hunk 4]
```diff
@@ -47,10 +47,5 @@
     ],
     "*.{jsx,js}": "eslint --cache --fix",
     "*.{js,css,md}": "prettier --write"
-  },
-    "husky": {
-    "hooks": {
-      "pre-commit": "lint-staged"
-    }
```
Comment (saifaldin) [inline package.json]: Please undo this deletion

**Review by saifaldin — DISMISSED** [review]

File: package.json (hunk 4 above)
Comment (EmanFouad) [reply package.json]: @seifeldeen3388 
it doesn't enable me to push my work

Comment (EmanFouad) [reply package.json]: img2

Comment (saifaldin) [reply package.json]: Yes, delete it without committing.
Just temporarily until the issue is fixed.

**Comment by Safei-Ashraf** [comment]
1-Tabs checkmark does not match the design
**Figma:**

image
**Implementation**

image
I think changing the white ball size could solve the inner checkmark, using height and width for it instead of padding worked for myself, feel free to fix it your way 

On this element padding does not match design 
image


-You might wanna consider adding proper cursor behaviour on any interactive elements in the future, this is not crucial at this point since we are just matching the static design, I think.

**Review by Safei-Ashraf — CHANGES_REQUESTED** [review]
Please consider changes in my previous comment.

**Review by Safei-Ashraf — APPROVED** [review]
LGTM

**Pull request merged**